shell/platform/windows/uwp_flutter_window.h

```diff
@@ -0,0 +1,176 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_UWP_FLUTTER_WINDOW_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_UWP_FLUTTER_WINDOW_H_
+
+#include "flutter/shell/platform/windows/flutter_windows_view.h"
+#include "flutter/shell/platform/windows/public/flutter_windows.h"
+#include "flutter/shell/platform/windows/uwp_game_pad.h"
+#include "flutter/shell/platform/windows/window_binding_handler.h"
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+#include <winrt/Windows.Gaming.Input.h>
+#include <winrt/Windows.Graphics.Display.h>
+#include <winrt/Windows.System.Profile.h>
+#include <winrt/Windows.UI.Input.h>
+#include <winrt/Windows.UI.ViewManagement.Core.h>
+#include <winrt/Windows.UI.ViewManagement.h>
+#include "winrt/Windows.System.Threading.h"
+#include "winrt/Windows.UI.Core.h"
+
+#include <windows.ui.core.h>
+#include <winrt/Windows.UI.Composition.h>
+#include <winrt/Windows.UI.Core.h>
+
+namespace flutter {
+
+struct WinRTWindowBounds {
+  float width;
+  float height;
+};
+
+// Implements a UWP CoreWindow.  Underlying window has been created and provided
+// by the runner.
+//
+// Specifically handles window events within windows.
+class WinRTFlutterWindow : public WindowBindingHandler {
+ public:
+  explicit WinRTFlutterWindow(ABI::Windows::UI::Core::CoreWindow* window);
+
+  virtual ~WinRTFlutterWindow();
+
+  // |SetView|
```

Comment:
These should be the name of the subclass they are overriding, not the method.

Reply:
Fix in next commit